internal/pkg/daemon/enricher/container.go

```diff
@@ -31,6 +31,11 @@ const (
 	crioPrefix = "cri-o://"
 )
 
+// NOTE(jaosorior): Should this actually be namespace-scoped?
```

Comment:
Unfortunately, this can't be namespace-scoped otherwise we won't be able to translate PID to container/pod across the cluster. Unless we find a better approach to it.